src/DonorDashboards/App.php

```diff
@@ -36,6 +36,22 @@ public function getOutput( $attributes ) {
 			$url = $url . '&accent-color=' . urlencode( $attributes['accent_color'] );
 		}
 
+		if ( isset( $_GET['give_nl'] ) ) {
+			$url = $url . '&give_nl=' . urlencode( give_clean( $_GET['give_nl'] ) );
+		}
+
+		if ( isset( $_GET['_give_hash'] ) ) {
+			$url = $url . '&_give_hash=' . urlencode( give_clean( $_GET['_give_hash'] ) );
+		}
+
+		if ( isset( $_GET['action'] ) ) {
+			$url = $url . '&action=' . urlencode( give_clean( $_GET['action'] ) );
+		}
+
+		if ( isset( $_GET['give_nl'] ) ) {
```

Comment:
This looks like a duplicate parameter, see like 39.

Also, we should consider building the URL using `add_query_arg` - passing it an array of key/value pairs.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5766/commits/bc459e152f089b37f116f722ed4b52d836c2fae5